lib/employee_crud.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy.orm import Session
```

Comment:
really should just focus on the appointment table. In the future, if you have a dependency on another team, reach out to them and let them know you need their code in order to complete yours. Work together with your team(s).